### PR TITLE
Removing doc code relating to MDN.

### PR DESCRIPTION
**Pull request by RedBrogdon**
* Removes some methods that used to grab MDN links on the fly for documentation
* Updates the Code responsible for generating HTML versions of docs provided by the analysis server for display.
* Removes related tests.

### CODE REVIEW

**Comment by RedBrogdon** [comment]
CC @miquelbeltran, since this is currently blocking dart-pad deploys and therefore his work.

File: test/web.dart [hunk 1]
```diff
@@ -6,7 +6,6 @@ library dart_pad.web_test;
 
 import 'core/dependencies_test.dart' as dependencies_test;
 import 'core/keys_test.dart' as keys_test;
-import 'documentation_test.dart' as documentation_test;
```
Comment (domesticmouse) [inline test/web.dart:9]: Should we also delete `documentation_test.dart` in this PR?

**Review by domesticmouse — APPROVED** [review]

File: test/web.dart (hunk 1 above)
Comment (parlough) [reply test/web.dart:9]: He already did so and I think it's fine to remove it.

**Review by parlough — APPROVED** [review]

**Comment by RedBrogdon** [comment]
I started this change this afternoon, and wasn't expecting anyone else to be working on it. Since @parlough's PR came in first, we should land that one instead.

**Pull request closed without merging**